package.json

```diff
@@ -43,6 +43,10 @@
     "aegir": "^12.0.8",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-webworker": "^1.3.0",
+    "mocha": "^5.0.1",
+    "multihashing": "^0.3.2",
```

Comment:
We only use `multihashing-async` to use WebCrypto in the browser.

Reply:
I was "experimenting", and likely I forgot to remove this package after (accidentally) adding it. I'll remove it.

Reply:
Oh, I must have misunderstood your comment. I'll remove `multihashing-async`, but I will still need `multihashing` in dev-dependencies. I use `multihashing` in my test case `cidperf-a.js`.